### ISSUE TITLE
Text of large font size in the editor not displaying properly on mobile devices

### ISSUE THREAD

**Issue by hafniz** [bestony/logoly#29]
Typing in long words or enlarging font size in the editor will result in wrapping of text in the preview, as shown in the screenshot. 
![Screenshot_20190327-124308](https://user-images.githubusercontent.com/42380178/55051188-12f37000-508f-11e9-940e-ece70248d432.png)

**Comment by bestony** [comment]
yes, I will build a new Version with mobile support today and tomorrow. 

Then, will be great.
